source/text/label.ts

```diff
@@ -17,10 +17,13 @@ import { Text } from './text';
 export class Label {
 
     /** @see {@link text} */
-    protected _text: Text | string;
+    protected _text: Text;
 
-    /** @see {@link wordWrap} */
-    protected _wordWrap = false;
+    /** @see {@link wordWrapper} */
+    protected _wordWrapper: Label.WordWrapper = Label.WordWrapper.None;
+
+    /** @see {@link ellpsisChars} */
```

Comment:
Type here, too.

Reply:
haha, thank you. Yes, this is a copy-paste-typo :D